### PR TITLE
json rule configuration documentation for common module types

### PR DESCRIPTION
**Pull request by gguba**
Signed-off-by: Georg Guba georg.guba@qaware.de

### CODE REVIEW

File: docs/documentation/development/rules/rule-configuration.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      "configuration": {
+        "itemName": "switchB",
+        "command": "OFF"
+      },
+      "type": "ItemPostCommandAction"
+    }
+
+The string used as the command depends on the item type and its corresponding supported command types, e.g. an HSB value of `120,100,100` to set a colored light's color to green. Similar to state change triggers, the correct state/action type is chosen automatically.
+
+## Item Types
```
Comment (kaikreuzer) [inline docs/documentation/development/rules/rule-configuration.md]: This table duplicates https://www.eclipse.org/smarthome/documentation/concepts/items.html - probably better to only link to that.

File: docs/documentation/development/rules/rule-configuration.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+| Dimmer | OnOff, IncreaseDecrease, Percent |
+| Group | |
+| Number | Decimal |
+| Player | PlayPause, NextPrevious, RewindFastforward |
+| Rollershutter | UpDown, StopMove, Percent |
+| String | String |
+| Switch | OnOff |
+
+
+## State and Command Type Formatting
```
Comment (kaikreuzer) [inline docs/documentation/development/rules/rule-configuration.md]: This is very useful information, but not specific to rules.
I'd suggest to add this to https://www.eclipse.org/smarthome/documentation/concepts/items.html instead, would that make sense?

File: docs/documentation/development/rules/rule-configuration.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+### HSBType
+
+HSB string values consist of three comma-separated values for hue (0-360�), saturation (0-100%), and value (0-100%) respectively, e.g. `240,100,100` for blue.
+
+### PointType
+
+`PointType` strings consist of three `DecimalType`s separated by commas, indicating latitude and longitude in degrees, and altitude in meters respectively. 
+
+### Enum Types
+| Type | Supported Values |
```
Comment (kaikreuzer) [inline docs/documentation/development/rules/rule-configuration.md]: please add an empty line here

**Comment by kaikreuzer** [comment]
Many thanks @gguba and sorry for taking so long to review it!

@danchom Please also feel free to review and comment.

File: docs/documentation/development/rules/rule-configuration.md (hunk 3 above)
Comment (danchom) [inline docs/documentation/development/rules/rule-configuration.md]: Looks great for me.

**Comment by danchom** [comment]
Looks great for me.

**Comment by kaikreuzer** [comment]
@gguba Did you see my comments?

**Comment by gguba** [comment]
@kaikreuzer I Incorporated the changes you suggested.

eclipse.org still won't activate my account so there will be no CLA.

**Comment by kaikreuzer** [comment]
> eclipse.org still won't activate my account so there will be no CLA.

Why is that? Is there any way that I can help? Without a CLA, we are not allowed to merge.

**Comment by gguba** [comment]
I've tried to register three times with eclipse.org within the last two months, clicking the activation link in the mail and waiting for activation. Nothing. I can't login. I can try again later though. Maybe it's fixed now.

**Comment by gguba** [comment]
There we go. It worked finally. CLA Signed.

**Comment by kaikreuzer** [comment]
Indeed, I can confirm the valid CLA. Thanks!

**Pull request merged**